base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
+
+**Document:**
+
+* {final-document}.md
+
+## Microcontent
+
+_TBD: We need to decide on the correct term to use between: structured data, microcontent, micro-content, microdata, snippets, metadata._
+
+* _Microcontent _
+    * _This is quite an evocative term._
+    * _I [notice](https://www.youtube.com/watch?v=CoShm-AXYP4&list=PLZAeFn6dfHpl2E5JhVd34llZD4a4oAeCo&index=19) Scott DeLoach is using the term "microcontent", but haven’t seen that term used elsewhere._
+    * _Searching “microcontent” in Google doesn’t find an authoritative definition._
+* _Microdata:_
+    * _Also evocative of the meaning._
+    * _Probably don’t use [microdata](https://www.w3.org/TR/microdata/), which is the term used for a specific W3C implementation for structured data._
+* _[Structured data](https://developers.google.com/search/docs/guides/intro-structured-data):_
+    * _Term is used by Google._
+    * _But isn’t as evocative as “microcontent”_
+    * _Doesn’t have a wikipedia entry_
+
+{Template author tip: All templates for The Good Docs Project must describe recommended [structured data](https://developers.google.com/search/docs/guides/sd-policies) fields. Fields selected should align with equivalent doctypes used in [schema.org](https://schema.org/).}
+
+Microcontent fields are machine readable snippets of [structured data](https://developers.google.com/search/docs/guides/sd-policies) which describe their associated page. It is typically presented as concise text or an image.
+
+All technical documentation should include microcontent. It helps search engines index pages appropriately, and facilitates content reuse.
+
+Within The Good Docs Project, we have adopted the [JSON-LD](http://json-ld.org/) format for microcontent, [as recommended by Google](https://developers.google.com/search/docs/guides/intro-structured-data).
+
+Note:
+
+* Many microcontent fields need to be maintained manually which adds a maintenance burden to a project.
+* If any of the microcontent on a site is inaccurate or dated, it will reduce the trustworthiness of all the microcontent on the site, as the user typically won’t be able to determine which of the content is trustworthy.
+* As such, a project should only adopt microcontent fields which are automatically maintained by your documentation publishing system, or where there is an established, low-effort process to ensure the microcontent is maintained.
+
+## Primary goal
+
+{Template author tip: Concisely state the primary purpose of this doctype, ideally in one short sentence.}
+
+## Target audience and user stories
+
+{Template author tip: You should document the key user stories that this template is likely to be used for.}
+
+{Docset owner tip: Select the user stories which are applicable to your project.}
+
+This {doctype} template aims to address the following user stories:
+
+* As a {user persona}, I want to {achieve a specific goal}, so that I can {achieve a larger business objective}.
```

Comment:
Cameron, I suspect this is particularly where you want my input. Let's have a conversation about where the emotional persona element could intersect here, and if there are other ways to construct user stories.

Reply:
As discussed, I'm looking for @ryanmacklin , Chris Ward, Aaron Peters, and others to take the lead on fleshing out this section in further iterations. I'll close this comment for the moment on the understanding that it will be addressed in a separate thread.